react/MuiCozyTheme/Switch/Readme.md

```diff
@@ -0,0 +1,15 @@
+```
+const React = require('react')
+const MuiCozyTheme = require('..').default
+const Switch = require('@material-ui/core/Switch').default;
+
+<MuiCozyTheme>
+  <Switch color="primary" defaultChecked />
+  <Switch color="secondary" defaultChecked />
+  <Switch color="primary" disabled />
+  <Switch color="secondary" disabled />
+</MuiCozyTheme>
+
+```
```

Comment:
Could you add a small description in front of each `Switch` ? It looks a bit empty when looking at the styleguidist. A link to the original component would be great also :)

```
<pre>color="primary" defaultChecked</pre>
<Switch color="primary" defaultChecked />
```

<img width="257" alt="image" src="https://user-images.githubusercontent.com/465582/51386906-62b03b00-1b24-11e9-8a16-f778e9de3b5f.png">